crypto_box/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![no_std]
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
It looks like you aren't actually using any `std`-dependent features but instead doing this to access `alloc::vec::Vec` which still works in a `no_std` context.

Instead of this, you can add an `alloc` feature instead, and use:

```rust
#[cfg(feature = "alloc")]
extern crate alloc;

#[cfg(feature = "seal")]
use alloc::vec::Vec;
```